spring-cloud-stream-binder-kinesis-core/pom.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.springframework.cloud</groupId>
+		<artifactId>spring-cloud-stream-binder-kinesis-parent</artifactId>
+		<version>1.0-SNAPSHOT</version>
+	</parent>
+	<artifactId>spring-cloud-stream-binder-kinesis-core</artifactId>
+	<description>Spring Cloud Stream Kinesis Binder Core</description>
+	<url>http://projects.spring.io/spring-cloud</url>
+	<properties>
+
+	</properties>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.springframework.cloud</groupId>
+			<artifactId>spring-cloud-stream</artifactId>
+		</dependency>
+
```

Comment:
We need 
```
<dependency>
    		<groupId>com.amazonaws</groupId>
    		<artifactId>aws-java-sdk-kinesis</artifactId>
</dependency>
```
otherwise who is going to provision us?..